website/cue/reference/components/transforms/filter.cue

```diff
@@ -67,7 +75,7 @@ components: transforms: filter: {
 		{
 			title: "Drop debug logs"
 			configuration: {
-				condition: #".level == "debug""#
+				condition: #".level != "debug""#
```

Comment:
👍

Reply:
Hehe, yeah, this was carried over from @leebenson's PR